lisa/tools/python.py

```diff
@@ -76,7 +72,7 @@ def install_packages(self, packages_name: str, install_path: str = "") -> None:
         if 0 != cmd_result.exit_code and get_matched_str(
             cmd_result.stdout, self._no_permission_pattern
         ):
-            cmd_result = self.run(cmd_line, update_envs=envs, sudo=True)
+            cmd_result = self.run(cmd_line, update_envs=envs)
```

Comment:
The sudo should be kept, because it's next try, if permission is required.

Reply:
I add chmod 777 in get_working_path_with_required_space and tested, all good

Reply:
If so, the Ln72 - 75 should be removed. But you can check on code history to know which distro need the sudo.

Reply:
oh oh, you are right, we can't assume this method always is invoked after get_working_path_with_required_space